### DISCUSSION TITLE
grafana panel dynamic show based the variables

### DISCUSSION THREAD

**Discussion by minicloudsky** [grafana/grafana#49563] [Dashboards, Panels & Visualizations]
**What happened**:
I want to implement grafana panel dynamic show based the variables, for example,The variable is ok, then show the below panel,if the variable is error, does not show the following panel
**What you expected to happen**:
grafana panel dynamic show based the variables, for example,The variable is ok, then show the below panel,if the variable is error, does not show the following panel
**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8
- Data source type & version: clickhouse
- OS Grafana is installed on: centos
- User OS & Browser: mac
- Grafana plugins:
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
